app/uk/gov/hmrc/eoricommoncomponent/frontend/views/eori_enrol_success.scala.html

```diff
@@ -43,8 +43,13 @@ <h2 class="govuk-heading-m">@messages("cds.subscription.outcomes.steps.next")</h
 
         @helpers.helpAndSupport()
 
-        @callBackContinue(service)
+        @if(service.callBack.nonEmpty){
+            @callBackContinue(service)
+
+            @feedback_di()
```

Comment:
Why do we duplicate @feedback_di()? Can't we just remove the else statement?

Reply:
Only reason I was using `else` part here is to follow conventional Scala style; i.e. in Scala code `if(...)` without `else` produces inference problem where return type of such expression is undefined (or `Any` type). 

Being this Twirl template I agree this can be be changed to get rid of `else` part.